libcudacxx/include/cuda/__memory_resource/properties.h

```diff
@@ -42,6 +42,11 @@ struct device_accessible
 struct host_accessible
 {};
 
+/// \struct managed_memory
+/// \brief The \c managed_memory property signals that the allocated memory is managed
+struct managed_memory
```

Comment:
I think this change requires changes to the documentation. 
https://nvidia.github.io/cccl/libcudacxx/extended_api/memory_resource/properties.html

Specifically: 

> For now, libcu++ provides two commonly used properties: `cuda::mr::device_accessible` and `cuda::mr::host_accessible`.